### ISSUE TITLE
Defect-level-3 [SCREEN READER]: File for disability compensation wizard contingent information presented not spoken by screen readers

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#22612]
Label: 22377

# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Issue Title: Defect level 3, File for disability compensation wizard contingent information presented not spoken by screen readers.

Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect information which comes up when I trigger an action to be consistent throughout the interaction. I also expect audible feedback when information pops up as a result of my changing options on a form.

<h2>Details</h2>
When a user responds that they are on active duty and selects their date of discharge, their screen reader speaks an alert when information pops up concerning eligibility for disability benefits. This allows the user to quickly ascertain whether they are eligible to apply and correct errors when they happen. However, if the user responds that they are not on active duty, they are presented with a second question. Information pops up based on their answer, but no alert is spoken. There is no submit button to trigger change. This could cause confusion for screen reader users who might not realize that they need to arrow down to find the information. 

Acceptance Criteria

- [x] Include Aria messages whose text is the information displayed which are triggered by the user's response to the second question.

Environment

<h2>Steps to Recreate</h2>
<ul>
<li>Access:

https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction
</li>

<li>Using JAWS or NVDA, respond no to the question "Are you on active duty."</li>
<li>Now, change the response to the question "Are you filing a new claim or disagreeing with a decision." </li>
<li>Note that you can arrow down and read the information which comes up in each case, but there is no audible indication when you make the change that new information is available.</li>
</ul>

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by Mottie** [comment]
@AngelaFowler82 The fix for this issue has just been merged into master and should be available in staging later today. Please let me know if you encounter any issues. Thanks!

**Comment by 1Copenut** [comment]
@AngelaFowler82 This issue appears to be remediated. Can we move this one to closed?

**Comment by AngelaFowler82** [comment]
Absolutely.
